treescript/versionmanip.py

```diff
@@ -87,7 +88,10 @@ def _get_version(file):
             replace_ver_in_file(file=abs_file,
                                 curr_version=curr_version, new_version=next_version)
     if changed:
+        dontbuild = get_dontbuild(context.task)
+        dontbuild = dont_build_check(dontbuild)
```

Comment:
In this section, a few things.

Lets not reuse the same variable name for two reasons, (dontbuild assigned to twice) especially when its type would be different. Makes it harder to reason about what is going on.

That said, if we get rid of the dont_build_check call (which you wrote to give us a " DONTBUILD" string, we can instead make this a file-level-constant of ``DONTBUILD_MSG = " DONTBUILD"`` (in the utils file) and then on the line following the 'Automatic version bump' message:
```
if dontbuild:
    commit_msg += DONTBUILD_MSG
```

We would make this same sort of change for the tagging line section above.